tasks/interactive-bots.md

```diff
@@ -233,10 +233,10 @@ doing this setup process. Add your notes as a `notes.md` file to
 
 ### Task Type C: Create an interactive bot
 
-We recommend claiming one of these tasks after completing Task Type A
+We recommend claiming this task type after completing Task Type A
 and Task Type B.
 
-Let *feature* be the feature mentioned in the task that brought you here.
+Let **feature** be the feature mentioned in the task that brought you here.
```

Comment:
@abhayKashyap03 @TomaszKolek This is also not right, since we're using `*feature*` later in the document.